### DISCUSSION TITLE
Cross-platform synchronization for Beatmaps, skins, etc.

### DISCUSSION THREAD

**Discussion by Hissymaster** [ppy/osu#27878] [Ideas] [answered]
I understand if you do not want to implemement this, due to the server overhead that may be needed. but it would make the experience of going between platforms (Android to PC, Pc to another PC, and vice versa and so on) much more seamless.

**Comment by smallketchup82** [comment, marked as answer]
https://github.com/ppy/osu/discussions/27376#discussioncomment-8585114
This has been suggested many times before, it's pretty low priority, might not get added at all. I'd recommend using something like [syncthing](https://github.com/syncthing/syncthing) to sync your osu folder

**Reply by Hissymaster** [reply]
Sorry.
